packages/dom/src/utils.js

```diff
@@ -41,3 +40,7 @@ export function styleSheetFromNode(node) {
 
   return sheet;
 }
+
+function rewriteLocalhostURL(path) {
+  return new URL(path, document.URL.replace(/(http[s]{0,1}:\/\/)localhost[:\d+]*/, '$1render.percy.local')).toString();
```

Comment:
```suggestion
  let baseURL = document.URL.toString();
  return baseURL.replace(/(http[s]{0,1}:\/\/)localhost[:\d+]*/, '$1render.percy.local').;
```

we can just do this.